interfaces/builtin/joystick.go

```diff
@@ -0,0 +1,83 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+)
+
+const joystickConnectedPlugAppArmor = `
+# Description: Allow reading and writing to joystick devices (/dev/input/js*).
+
+/dev/input/js[0-9]* rw,
```

Comment:
Looking a thttps://github.com/torvalds/linux/blob/master/Documentation/admin-guide/devices.txt, it will always have a number, so the current rule is correct.

In reading that, please also add this for futureproof-ness:
```
/run/udev/data/c13:{[0-9],[12][0-9],3[01]} r,
```